vkext/vkext.cpp

```diff
@@ -614,11 +653,12 @@ PHP_FUNCTION (vk_utf8_to_win) {
 PHP_FUNCTION (vk_win_to_utf8) {
   char *text;
   long text_len = 0;
-  if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "s", &text, &text_len) == FAILURE) {
+  bool escape = false;
+  if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "s", &text, &text_len, &escape) == FAILURE) {
```

Comment:
Now it still accepts only one parameter and will fail with error if you pass the second one. Like this:
```
Warning: vk_win_to_utf8() expects exactly 1 parameter, 2 given in /home/denis/kphp/dev/index.php on line 3
```
You need to add some type specifiers to fix this. You can find them [here](http://php.adamharvey.name/manual/ru/internals2.funcs.php). I think it should look smth like this:
```suggestion
  if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "s|b", &text, &text_len, &escape) == FAILURE) {
```